File: scala/FizzBuzz.scala

object FizzBuzz {

	def main(args: Array[String]) {
	
		var i = 1;
		
		for( i <- 1 to 100 ) {
			
			var three = i % 3;
			var five  = i % 5;
			
			if( three == 0 && five == 0 ) {
				println("FizzBuzz");
			}
			else if ( three == 0 ) {
				println("Fizz");
			}
			else if ( five == 0 ) {
				println( "Buzz" );
			}
			else {
				println(i);			
			}
			
		}
	
	} // main

} // class

File: ruby/fizzbuzz.rb
#!/usr/bin/ruby

$i    = 1
$num  = 100

while $i <= $num  do

  $three = $i % 3
  $five  = $i % 5

  if $three == 0 and $five == 0
    puts "FizzBuzz"
  elsif $three == 0
    puts "Fizz"
  elsif $five == 0
    puts "Buzz" 
  else 
    puts $i
  end

  $i +=1
   
end